test/models/work_test.rb

```diff
@@ -0,0 +1,71 @@
+require "test_helper"
+
+describe Work do
+  # before do
+  #   @post = Post.create!(name:'Awesome Post')
+  # end
+  before do
+    @work = works(:four_hour_chef)
+  end
+
+  describe 'validations' do
+    it 'has a title' do
+      book_title = @work.title
+      book_title.must_equal("The 4-Hour Chef")
```

Comment:
Having a title isn't really testing the validation - we want to create it with a title and ensure that it is `valid?`, and then do the same  for the inverse scenario (like you are doing below)